CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+* @generalassembly/infrastructure
+
+# Base config files
+dev_tools/rubocop/base.yml @generalassembly/infrastructure @generalassembly/edtech @generalassembly/martech @generalassembly/engineering-support
+dev_tools/rubocop/rails-base.yml @generalassembly/infrastructure @generalassembly/edtech @generalassembly/martech @generalassembly/engineering-support
+dev_tools/rubocop/rspec-base.yml @generalassembly/infrastructure @generalassembly/edtech @generalassembly/martech @generalassembly/engineering-support
```

Comment:
I think you can combine this into a single rule where the teams own everything in the `rubocop` directory and its subdirectories by default:
`/dev_tools/rubocop/ @generalassembly/infrastructure @generalassembly/edtech @generalassembly/martech @generalassembly/engineering-support`

The team-specific rules are listed after this so they'll take precedence for those matching files.

We can also test this out before merging but the test PR needs to be compared against this branch instead of `master` in order for the rules to be applied.

Reply:
I think I'll leave this for a post-merge cleanup now. Nothing wrong with being explicit on the first pass, until we're sure :)